### PR TITLE
ecshreve: some simple cleanup

### PR DESCRIPTION
**Pull request by ecshreve**
This PR does some basic cleanup.
- change some of the error checks to be inline `if err := someFunc(); err != nil {`
- tweak some comments for readability and godoc conventions
- regenerate docs

This should be a no-op, but I did sanity check by running locally.

### CODE REVIEW

**Review by brittonhayes — APPROVED** [review]
Love it! 😊 Thanks for the welcome contribution homie.

**Comment by ecshreve** [comment]
@brittonhayes feel free to merge whenever you have time, I don't have write access so can't merge it myself :)

**Pull request merged**
